clothing_app/clothing_app/middlewares.py

```diff
@@ -100,3 +102,49 @@ def process_exception(self, request, exception, spider):
 
     def spider_opened(self, spider):
         spider.logger.info("Spider opened: %s" % spider.name)
+
+
+class ClothingAppProxyMiddleware(object):
+    def __init__(self):
+        self.proxy = None
+        self.proxy_manager = ProxyManager()
+
+        self.proxies = [
+            Proxy("http://stylesage:O5s6a9jaAs@66.56.67.157:60000"),
```

Comment:
We should not write proxies OR any credentials/configurations in the code. It should be in separate files. So, create a file `proxies.txt` in some folder and add its path in the settings file. Use that path in the middleware to read the file and load the proxies.